### ISSUE TITLE
How to set from which point in width the sliding pane should begin

### ISSUE THREAD

**Issue by shesellsanctuary** [DimitryDushkin/sliding-pane#62]
Hello, thank you for the library, it works very well. 

I am having trouble trying to set the 
`.slide-pane_from_left.content-after-open {
    transform: translateX(0%);
}`
to a different value. Is there a way to do it?

Thanks

**Comment by DimitryDushkin** [comment]
Hey! If you want different width, try ‘width’ property. 
Otherwise you can add custom class using ’className’ property and set style starting with this class name.

**Comment by shesellsanctuary** [comment]
Yes! I want to override the style, but I have not found where to put it exactly, because I see the react modal portal div class is outside the root.
